src/components/email/Email.tsx

```diff
@@ -70,11 +70,12 @@ export default class Email extends React.Component<
       this.siteConfig.defaultSubjectLine.trim()
     );
     const body = encodeURIComponent(this.emailData.body.trim());
-    const recipients = this.emailData.recipients.join(", ");
-    const cc = this.emailData.cc?.join(", ");
-    const ccText =
-      cc !== undefined && cc !== null && cc.length > 0 ? `cc=${cc}&` : "";
-    window.location.href = `mailto:${recipients}?${ccText}subject=${subject}&body=${body}`;
+    const recipients = this.emailData.recipients;
+    const cc = this.emailData.cc || [];
+    const bcc = recipients.concat(cc);
+    window.location.href = `mailto:?bcc=${bcc.join(
+      ","
+    )}&subject=${subject}&body=${body}`;
```

Comment:
w/out recipients stll works - https://stackoverflow.com/questions/23813123/how-can-i-create-mailto-links-without-recipient-that-will-work-on-chrome-on-ubun

Also tested.. for raleigh which has a cc

![image](https://user-images.githubusercontent.com/2534903/84924671-1bfea180-b086-11ea-800e-81ed4b9c270c.png)